src/docs/product/error-monitoring/reprocessing/index.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Reprocessing
+sidebar_order: 10
+description: Learn about reprocessing errors with new debug files and sourcemaps.
+---
+
+After you have uploaded new [debug information
+files](/platforms/native/data-management/debug-files/) or
+[sourcemaps](/platforms/javascript/sourcemaps/), Sentry only uses them for new
+events going forward. _Reprocessing_ allows you to apply them to older events as
+well.
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+After becoming an Early Adopter, you will find a new button when viewing an issue:
+
+!["Reprocess this issue" button next to Resolve, Ignore, Share, Delete actions in issue details page](reprocessing-btn.png)
+
+The dialogue behind it asks you two questions:
```

Comment:
after some internal debate (and noting that dialog is not in our Style Guide, so will be added):
```suggestion
The dialog requires two answers:
```